teamfisk/TacticalZ
# Collision in test phase

The collision is not perfect but is in a usable state. It has not been tested over network yet.

I'd like anyone who has the time, to pull this branch and test if the collision works (or if there is any bugs) and if it feels right in the game (or if it should be enhanced somehow). Tell me and/or make a comment/issue.

To test:
Under [Bindings] in Config.ini set:
1=PickTeam,1
2=PickTeam,2
3=PickTeam,3

Import GameMap.xml to test the collision in game. You are going to want to play in Release mode.
Unpause the world in the editor, then you should be able to do F1 to disable editor, and 2 or 3 to spawn as a player.
